WordPress/src/main/java/org/wordpress/android/ui/comments/unified/EditCancelDialogFragment.kt

```diff
@@ -0,0 +1,53 @@
+package org.wordpress.android.ui.comments.unified
+
+import android.app.Dialog
+import android.os.Bundle
+import androidx.fragment.app.DialogFragment
+import androidx.lifecycle.ViewModelProvider
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import javax.inject.Inject
+
+class EditCancelDialogFragment : DialogFragment() {
+    @Inject lateinit var viewModelFactory: ViewModelProvider.Factory
+
+    private var viewModel:UnifiedCommentsEditViewModel? = null
```

Comment:
A space before the type should fix the lint issue ;)
```suggestion
    private var viewModel: UnifiedCommentsEditViewModel? = null
```

Reply:
I was missing the lint failure, thanks sir 😄